packages/cozy-intent/package.json

```diff
@@ -45,7 +48,8 @@
     "build:watch": "yarn build --watch",
     "lint": "eslint . --ext .ts,.tsx",
     "lint:fix": "yarn lint --fix",
-    "prepublishOnly": "yarn build"
+    "prepublishOnly": "yarn build",
+    "test": "jest"
```

Comment:
clap clap 👏

Reply:
I forgot to change the PR target after a rebase. This comment is related to : https://github.com/cozy/cozy-libs/pull/1438